src/globals/scss/settings/_measurements.scss

```diff
@@ -8,7 +8,8 @@ $govuk-gutter-half: $govuk-gutter / 2;
 $govuk-border-width-mobile: 4px;
 $govuk-border-width-tablet: 5px;
 $govuk-border-width-form-element: 2px;
-$govuk-border-width-error: 4px;
```

Comment:
We've got so many border widths being defined here, and I'm not sure where they're all used. It might be good to sit down with Dave and try and work out whether we can simplify this.

Reply:
PR here: https://github.com/alphagov/govuk-frontend/pull/524

Reply:
Sorry, my original comment wasn't very clear… I was thinking more about trying to avoid introducing yet another variable than about removing any of the existing ones. For example, can the `$govuk-border-width-error-field-group` be avoided by just using `$govuk-border-width`? Or, if we think this variable does need to exist, can we do `$govuk-border-width-error-field-group: $govuk-border-width`?

(Also, can we name `$govuk-border-width-error-field` something like `$govuk-border-width-form-element-error` to make it clear it's the width of the element not the width of the left-hand border? It took me a couple of readings of the code to realise that it was related to the thickness of the border on the input rather than the thickness of the left-hand border.)

Reply:
Ahh cool. I'll look into this now.

Reply:
I've removed the tablet variable, renamed `$govuk-border-width-error-field` (that was a good call) and `$govuk-border-width-form-group-error` now uses `$govuk-border-width`.